### PR TITLE
add html.js v0.12.1 

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/nbubna/HTML
**Watch 69
  Star 1,411
 Fork 112**
@Amomo  could u help me check this PR for #6587 **ASAP**? thank you!

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/html.js/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "HTML.base*.js",
+        "HTML.core*.js",
+        "HTML.event*.js",
+        "HTML.js",
+        "HTML.min.js"
+      ]
```
Comment (PeterDaveHello) [inline ajax/libs/html.js/package.json]: Why not `"*.js"`?

**Comment by maruilian11** [comment]
@PeterDaveHello i have added `*.js` files.

**Comment by PeterDaveHello** [comment]
Don't you need to minify all the files?

**Comment by maruilian11** [comment]
@PeterDaveHello sorry, added them.

**Pull request merged**